content/geometry/object-human/human-structure.adoc

```diff
@@ -8,36 +8,36 @@ legend
 |_ [[../geometry/object-human/human-index.html#_grp_root Grp_Root]]
   |_ [[../geometry/object-human/human-index.html#_armature Armature]]
```

Comment:
Here in the structure it is just called Armature, but the section headline is Armature_<Name>. The name however is not specified there. So should we just call it Armature without any suffix?

Reply:
I would add and declare it as an optional postfix, but as recommend if multiple human assets are prepared in one 3D scene.